subsys/bluetooth/shell/rfcomm.c

```diff
@@ -33,7 +33,7 @@
 
 #define DATA_MTU 48
 
-NET_BUF_POOL_FIXED_DEFINE(pool, 1, DATA_MTU, 8, NULL);
+NET_BUF_POOL_FIXED_DEFINE(pool, 1, DATA_MTU, 0, NULL);
```

Comment:
are you using the BR/EDR part of the stack?

Reply:
Just modify it by the way.